pytorch_lightning/core/lightning.py

```diff
@@ -91,14 +91,9 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
         # torch/nn/modules/module.py#L227)
         torch._C._log_api_usage_once(f"lightning.module.{self.__class__.__name__}")
 
-        self._loaded_optimizer_states_dict = {}
```

Comment:
I recently formally deprecated this in https://github.com/PyTorchLightning/pytorch-lightning/pull/8229
Do you think it can be outright removed?